core/src/main/scala/org/apache/spark/ui/storage/RDDPage.scala

```diff
@@ -76,7 +76,8 @@ private[ui] class RDDPage(parent: SparkUITab, store: AppStatusStore) extends Web
         rddStorageInfo.partitions.get,
         blockPageSize,
         blockSortColumn,
-        blockSortDesc)
+        blockSortDesc,
+        store.executorList(false))
```

Comment:
Should this only list active executors?

Theoretically you shouldn't have any blocks stored on dead executors.

Reply:
As we have a fallback to executorId we can do that.